README.md

```diff
@@ -63,36 +74,76 @@ sim_log_data(path_to_synthetic_log_data)
+location = "GB/Bristol" # location of person to generate format: Country_2_letter_ISO_code/City_Name
+start_date = datetime.date(2021, 10, 1) # start date of generated trajectories
+end_date = datetime.date(2021, 10, 5) # end date of trajectories
+api_key = "mock_api_key" # api key for openroute service, generated from https://openrouteservice.org/
 cycle = 15 # Length of off-cycle + length of on-cycle in minutes
-p = 0.8 # Length off-cycle / (length off-cycle + length on-cycle)
-sim_GPS_data(cycle, p, path_to_synthetic_log_data)
+percentage = 0.8 # Length off-cycle / (length off-cycle + length on-cycle)
+attributes_dict = None # dictionary of personallity attributes for each user, set to None if random, check Attributes class for usage in simulate_gps_data module. Example of dictionary of user attributes:
+attributes_dict = {
+    "User 1":
+    {
+        "main_employment": "none", 
+        "vehicle" : "car",
+        "travelling_status": 10,
+        "active_status": 0
+    },
+
+    "Users 2-4":
+    {
+        "main_employment": "university",
+        "vehicle" : "bicycle",
+        "travelling_status": 8,
+        "active_status": 8,
+        "active_status-16": 2 
+    },
+
+    "User 5":
+    {
+        "main_employment": "office",
+        "vehicle" : "foot",
+        "travelling_status": 9,
+        "travelling_status-20": 1,
+        "preferred_exits": ["cafe", "bar", "cinema"] 
+    }
+}
+sample_gps_data = sim_gps_data(n_persons, location, start_date, end_data, api_key, cycle, percentage, attributes_dict)
 
 # 2. Specify parameters for imputation 
 # See https://github.com/onnela-lab/forest/wiki/Jasmine-documentation#input for details
 tz_str = "America/New_York" # time zone where the study took place (assumes that all participants were always in this time zone)
-option = "daily" # Generate summary metrics "hourly", "daily" or "both"
+frequency = Frequency.DAILY # Generate summary metrics Frequency.HOURLY, Frequency.DAILY or Frequency.BOTH
 save_traj = False # Save imputed trajectories?
+parameters = None # Hyperparameters class for imputation (default leave None)
+places_of_interest = ['cafe', 'bar', 'hospital'] # list of locations to track if visited, leave None if don't want these summary statistics
+save_log = False # True if want to save a log of all locations and attributes of those locations visited
+threshold = 15 # threshold of time spent in a location to count as being in that location, in minutes
+split_day_night = False # split summary statistics to day and nighttime statistics, only if frequency == Frequency.DAILY
+person_point_radius = 2 # meters around person to count as being in a place
+place_point_radius = 7.5 # meters around location to count as POI
 time_start = None 
 time_end = None
-beiwe_id = None
-parameters = None
+participant_ids = None
 all_memory_dict = None
-all_BV_set= None
+all_bv_set= None
+quality_threshold = 0.05 # minimum percentage of data being valid needed to perform the analysis
```

Comment:
Since this is just a simple example for getting started we should probably omit any values that are set by default in the function signature.